### ISSUE TITLE
Will there be low-poly characters?

### ISSUE THREAD

**Issue by FuzzyWobble** [animate1978/MB-Lab#341]
Characters are currently too high poly for game use.

**Comment by Hopefullyidontgetbanned** [comment]
even without subdivisions (MB-Lab automatically adds them, you can remove them before finalization), characters can fit within the poly budget of 7th generation video game consoles (12+ year old hardware) unless you are talking about mobile/web they are decently sized in terms of polycount

**Comment by FuzzyWobble** [comment]
Thanks for the response. It's creating characters ~17k polygons. Does that sound right or am I doing something wrong? As a comparison, a body from Sims4 is ~7.5k. For my use I am trying to get closed to 4k. I can always decimate, just curious if there was an option built into MBLab for lowpoly.

**Comment by Hopefullyidontgetbanned** [comment]
yes 17k is the base and future updates will likely be marginally larger/smaller but decimating it is always an option for games

**Comment by bmourit** [comment]
Yes. For mobile I manually decimated the mesh so I could keep the details I wanted. With my own eyes and hair from scratch, I got this down pretty easier to around 6-7k including everything.
I also made my own skin though and customized the mesh quite a bit by hand. The base mesh is still a great starting point though. You'll want to keep it anyway (perhaps an even higher poly one too) to bake the normals to the low poly. The results are excellent imo.
